tests/DCPS/Reconnect/.gitignore

```diff
@@ -19,9 +19,9 @@
 /subscriber_ready.txt
 /publisher_finished.txt
 /publisher_ready.txt
-/test.log
 /publisher1.log
 /publisher2.log
 /subscriber1.log
 /subscriber2.log
-
+/repo.log
```

Comment:
Not a big deal, but this should be covered by the wildcard ignore in the root .gitignore.  You can use the script tools/scripts/update_gitignore.pl to maintain these.  core could be covered by your individual git config.